publication/ver11/7-resources/validation/td-json-schema-validation.json

```diff
@@ -1,9 +1,9 @@
 {
   "title": "Thing Description",
-  "version": "1.1-09-November-2023",
+  "version": "1.1-12-March-2025",
```

Comment:
I strongly advise using [ISO8601](https://xkcd.com/1179/) date format (YYYY-MM-DD) for most if not all purposes.

Among other things, this ensures that any sortable (e.g., tabular) listing will always put "March" ("03") earlier than "December" ("12"), while the format used above will always put "July" ("07") before "June" ("06").

Reply:
I agree with the statement. I just wonder whether it has any *negative* effect since we were using the other format in the past 🤷‍♂️